cfe_internal/recommendations.cf

```diff
@@ -33,7 +33,7 @@ bundle agent postgresql_conf_reccomendations
 
       "calculated_shared_buffers_MB"
         string => format( "%d0",
-                          eval( "$(mem_info_data[MemTotal][1]) * 25 / 100 / 1024"));
+                          eval( "$(mem_info_data[MemTotal][1]) * 25 / 100 / 1024"), "math", "infix");
 
```

Comment:
What do changes in recommendations.cf related to _MB have to do with sles/suse classes?